psycop/common/feature_generation/data_checks/flattened/feature_describer_tsflattener_v2.py

```diff
@@ -0,0 +1,98 @@
+import re
+from dataclasses import dataclass
+from typing import Any, Callable
+
+import polars as pl
+
+
+@dataclass
+class ParsedPredictorColumn:
+    col_name: str
+    feature_name: str
+    fallback: str
+    time_interval_start: str
+    time_interval_end: str
+    time_interval_format: str
+    resolve_multiple_strategy: str
+    is_static: bool
+
+
+def tsflattener_v2_column_is_static(col_name: str) -> bool:
+    str_match = "within"
+    return str_match not in col_name
+
+
+def get_value_proportion(series: pl.Series, value: str) -> float:
+    """Get proportion of series that is equal to the value argument."""
+    if value == "nan":
+        return round(series.is_nan().mean(), 2)  # type: ignore
+    return round(series.eq(float(value)).mean(), 2)  # type: ignore
```

Comment:
what if fallback value is also a "naturally occurring" value in the data?

Reply:
we can't protect against this without seeing the raw data as well. I think it'll have to be a note in the table description